spring-cloud-common-security-config-web/src/main/java/org/springframework/cloud/common/security/OAuthSecurityConfiguration.java

```diff
@@ -141,12 +123,14 @@ protected void configure(HttpSecurity http) throws Exception {
 		final BasicAuthenticationEntryPoint basicAuthenticationEntryPoint = new BasicAuthenticationEntryPoint();
 		basicAuthenticationEntryPoint.setRealmName(SecurityConfigUtils.BASIC_AUTH_REALM_NAME);
 		basicAuthenticationEntryPoint.afterPropertiesSet();
-		final Filter oauthFilter = oauthFilter();
+		//final Filter oauthFilter = oauthFilter();
 		BasicAuthenticationFilter basicAuthenticationFilter = new BasicAuthenticationFilter(
 				providerManager(), basicAuthenticationEntryPoint);
-		http.addFilterAfter(oauthFilter, basicAuthenticationFilter.getClass());
-		http.addFilterBefore(basicAuthenticationFilter, oauthFilter.getClass());
-		http.addFilterBefore(oAuth2AuthenticationProcessingFilter(), basicAuthenticationFilter.getClass());
+//		http.addFilterAfter(oauthFilter, basicAuthenticationFilter.getClass());
```

Comment:
Can we delete these if we don't need the explicit configuration?

Reply:
Fixed